mlflow/store/db/utils.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+
+def _upgrade_db(url):
+    """
+    Upgrade the schema of an MLflow tracking database to the latest supported version.
+    version. Note that schema migrations can be slow and are not guaranteed to be transactional -
+    we recommend taking a backup of your database before running migrations.
+
+    :param url Database URL, like sqlite:///<absolute-path-to-local-db-file>. See
+    https://docs.sqlalchemy.org/en/13/core/engines.html#database-urls for a full list of valid
+    database URLs.
+    """
+    # alembic adds significant import time, so we import it lazily
```

Comment:
Copied this from Airflow: https://github.com/apache/airflow/blob/b69c686ad8a0c89b9136bb4b31767257eb7b2597/airflow/utils/db.py#L297